packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/intro.md

```diff
@@ -0,0 +1,5 @@
+To retain authentication control of your Angular single-page app (SPA) without redirection to Okta, you can implement the [embedded authentication](/docs/concepts/redirect-vs-embedded/#embedded-authentication) model with the help of Okta Auth JS and Okta Angular libraries. These libraries provide you with common, reusable OAuth 2.0 methods and properties to handle the interaction between the Okta authorization server and your client app. This allows you to customize the sign-in experience for your app with minimal use of low-level [Okta Authentication](/docs/reference/api/authn/) and [Okta OpenID Connect & OAuth 2.0](/docs/reference/api/oidc/) APIs.
```

Comment:
Only Okta Auth JS library is used for the example. The Okta Angular library wasn't used for this.

Reply:
Thanks, removed this bit and revised text.